lib/src/chart/line_chart/line_chart_data.dart

```diff
@@ -397,6 +401,9 @@ class LineChartBarData with EquatableMixin {
       isStepLineChart: b.isStepLineChart,
       lineChartStepData:
           LineChartStepData.lerp(a.lineChartStepData, b.lineChartStepData, t),
+      properties: {}
+        ..addAll(a.properties)
```

Comment:
Hi, sorry for my late reply.
You should not do this here. Because everytime you set a new value to the properties list, it adds to the previous map and it is not a correct behaviour.

Suppose that you have:
```dart
{
   "title": "Hello"
}
```

And in the next setState, you change it to 
```dart
{
  "name": "Hello"
}
```

And in the end you have:
```
{
  "title": "Help",
  "name": "Hello"
}
```
which is not correct in my opinion